project_parent_task_filter/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
Is needed?

Reply:
No. I was looking that right now, I could delete the models folder.
And the project_parent_task_filter/__init__.py should be empty or could be deleted?

Reply:
Remove model folder and empty project_parent_task_filter/init.py but don't remove it.